### PR TITLE
React Calculator Milestone 2: Components!

### PR DESCRIPTION
**Pull request by lmaldonadoch**
For this milestone, I created the components for the project. I validated the type and presence through propTypes and combined them together to display a calculator with no style or functionality to the DOM.

## Please note!
ESlynt was complaining about 2 violations. I disabled them and here are the reasons:

## Module-specific requirements.

- Make sure that you use props and state in a way they were designed for.
- Make sure that you understand the difference between stateful and stateless components. 

## Project Specific

- Use create-react-app to initialize a new React app
- Documentation: https://github.com/facebook/create-react-app
- Delete any unnecessary file or asset
- Add prop-types using npm for type checking props.
-Add big.js using npm. 
- Run the application with npm start and make sure that everything works correctly.
- Follow the instructions in create-react-app-buildpack to deploy the app to Heroku
  * Initialize a git repo
  * Create a Heroku app
  * Commit and deploy to Heroku
  * Make sure the Heroku app works correctly
  * Push everything to Github
  * Documentation: https://github.com/mars/create-react-app-buildpack

## General requirements:
- Add a descriptive README file to your repositories as described.
- Use meaningful names for your Pull Requests and add a summary of its content as described.
- Use descriptive commit messages as described.
- Do not commit files that are not related to the project. Instead, add them to .gitignore file.
- Do not commit inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

### Git Flow:
- Use a specific branch for each feature.
- Use a descriptive name for your branches.
- Use Rubocop to validate your code.

### CODE REVIEW

**Review by fatymahmed — CHANGES_REQUESTED** [review]
## Hi @lmaldonadoch,
Good job in this milestone  👏🏻👏🏻, very descriptive PR, however, there are a couple of things that we need to change.
**Please check my comments below for a detailed review** 

## Status: Changes Required ♻️
_Feel free to reach out to me on slack (Fatima) if you have any questions or need any help_

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    super(props);
+    this.state = {
+      result: '0',
+    };
+  }
+
+  render() {
+    const { result } = this.state;
+    return <div>{result}</div>;
+  }
```
Comment (fatymahmed) [inline src/components/Display.js]: **Display component**
- it should accept the result of the calculation as an incoming **prop** not as a **state**  see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png) of requirement
- the result prop should be a String with a default value of `"0"` (string)  [should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)   [default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)

Comment (lmaldonadoch) [reply src/components/Display.js]: Thank you for pointing it out! It has been fixed :)

**Review by shloch — APPROVED** [review]
## Hello @lmaldonadoch 

accomplished

Good to go, great job

**Pull request merged**